### DISCUSSION TITLE
Use the Plejd wireless switches

### DISCUSSION THREAD

**Discussion by cufk** [icanos/hassio-plejd#146] [General]
Hi,

Quite new to Plejd and their product line and how it all works. Would it be possible to:

1. Only use the Plejd wireless switches and use them in HASS as a regular wireless switch/remote?
2. Do the Plejd products work locally or do their products need the cloud to work? 

Thanks! :)

**Comment by faanskit** [comment]
1. Unlikely, and if so - suboptimal. hassio-plejd need an open Bluetooth communication path with a device. WPH/WRT are battery powered devices and will not keep the connection up. At least one Plejd device will be needed in your installation.
2. The devices are configured by the Plejd App. The Plejd stores the configuration in Plejds cloud. hassio-plejd needs to connect to the cloud at minimum when changes to the system has been made. After that the Plejd products are only talking over Bluettoth with each other.
3. Currently the only way to drive an automation is to connect the WPH01/WTR-01 to a Plejd device, and it has significant limitations. In the coming 8.0.0 release hopefully PR #188 will be included. PR #188 registers WPH-01/WRT-01 buttons as `device_triggers` to be used in automations in Home Assistant. This removes the need for a WPH-01/WRT-01 to be connected to a device as a workaround to drive automations.
